cypress/e2e/tests/migration/applicationinventory/analysis/gradle_analysis.ts

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    getRandomApplicationData,
+    deleteByList,
+    deleteAllMigrationWaves,
+    deleteApplicationTableRows,
+} from "../../../../../utils/utils";
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import { AnalysisStatuses } from "../../../../types/constants";
+import { Application } from "../../../../models/migration/applicationinventory/application";
+
+const applications: Analysis[] = [];
+
+describe(["@tier2"], "Gradle Analysis", () => {
+    before("Login", function () {
+        login();
+        deleteAllMigrationWaves();
+        deleteApplicationTableRows();
+    });
+
+    beforeEach("Load data", function () {
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+
+        cy.intercept("GET", "/hub/application*").as("getApplication");
+        Application.open(true);
```

Comment:
Is it necessary to use Application.open(true) here, since the application.create() function will already open the application page?

Reply:
You're right.

Right now it is not needed but I'm working already in the next test. 
The `application.create` method opens the app page but does not forces a reload, and that's something I want to do between tests to ensure the failure of one test doesn't leave the page in a weird state (like an open dialog) that could affect the next one.